model/src/main/java/hu/blackbelt/judo/meta/jsl/validation/JslDslValidator.xtend

```diff
@@ -818,15 +818,15 @@ class JslDslValidator extends AbstractJslDslValidator {
         }
     }
 
-    @Check
-    def checkForDuplicateNameForEntityMemberDeclaration(EntityMemberDeclaration member) {
-        if (member instanceof Named && (member.eContainer as EntityDeclaration).getMemberNames(member).map[n | n.toLowerCase].contains(member.name.toLowerCase)) {
-            error("Duplicate member declaration: '" + member.name + "'",
-                member.nameAttribute,
-                DUPLICATE_MEMBER_NAME,
-                member.name)
-        }
-    }
+//    @Check
+//    def checkForDuplicateNameForEntityMemberDeclaration(EntityMemberDeclaration member) {
+//        if (member instanceof Named && (member.eContainer as EntityDeclaration).getMemberNames(member).map[n | n.toLowerCase].contains(member.name.toLowerCase)) {
+//            error("Duplicate member declaration: '" + member.name + "'",
+//                member.nameAttribute,
+//                DUPLICATE_MEMBER_NAME,
+//                member.name)
+//        }
+//    }
```

Comment:
What is the reason of commented out check? Where is the alternative?

Reply:
The "def checkForDuplicateName(Named named)" changed to check any Named declaration.
The commented out code is temporary and will be removed later.